modules/terms/partials/topic-partition.adoc

```diff
@@ -0,0 +1,4 @@
+=== topic partition
+:term-name: topic partition
+:hover-text: A topic may be partitioned through multiple brokers. A "topic partition" represents this logical separation on Redpanda. It is managed by Raft, applying voting mechanisms such as Round Robin. In contrast, other Kafka applications use ZooKeeper to manage their partitions. In Redpanda this is managed natively by Raft. 
```

Comment:
```suggestion
:hover-text: A topic may be partitioned through multiple brokers. A "topic partition" represents this logical separation in Redpanda, which is managed by Raft and applies voting mechanisms such as Round Robin. In contrast, other Kafka applications use ZooKeeper to manage their partitions. In Redpanda this is managed natively by Raft. 
```

Reply:
@paulzhang97 Round Robin should not be capitalized

Reply:
Actually, for hover text, it may be best just to end this with "...natively by Raft." You can put in a line break and put the rest (about Zookeeper) below, so it shows in the glossary but not the hover text.